src/service/service-agreement-service.js

```diff
@@ -27,50 +27,6 @@ class ServiceAgreementService {
 
     async calculateScore(peerId, blockchainId, keyword, hashFunctionId) {
         const peerRecord = await this.repositoryModuleManager.getPeerRecord(peerId, blockchainId);
-        /* const keyHash = await this.validationModuleManager.callHashFunction(
```

Comment:
This part is also used in open [PR 2273](https://github.com/OriginTrail/ot-node/pull/2273), should we keep it? @zeroxbt

Reply:
yes, makes sense

Reply:
I'll revert these changes then